File: TaskManager/src/CommandManager.java
import java.util.Stack;

public class CommandManager {
    private Command command;

    public Stack<Command> executedCommands = new Stack<>();
    public Command UndoedCommand;

    public void setCommand(Command command) {
        this.command = command;
    }

    public Issue executeCommand(){
        executedCommands.push(command);
        return command.execute();
    }

    public Issue undoCommand(){
        UndoedCommand = command;
        return command.undo();
    }
}

File: taskManager/CommandManager.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace taskManager
{
    public class CommandManager
    {
        Stack<Icommand> executeCommands = new Stack<Icommand>();
        Stack<Icommand> unDoneCommands = new Stack<Icommand>();

        public void Execute(Icommand command)
        {

        }

    }
}
